ouroboros-consensus/docs/Onboarding.md

```diff
@@ -0,0 +1,144 @@
+ledger rules. We therefore work closely with the Network Team and the Ledger
+Team.
+
+The Network Layer is also defined in this repository. It manages the nodes'
+connections to its neighbors. So it ultimately provides communication channels
+to the Consensus Layer, while the Consensus Layer reports back to it if a
+neighbor has misbehaved etc. The Network Layer also provides the library used to
+define the Consensus Layer's _client_ and _server_ state machines that let each
+connected pair of nodes exchange messages according to the various _mini
+protocols_ (cf `typed-protocols` package).
+
+The ledger rules are defined in
+https://github.com/input-output-hk/cardano-ledger-specs. The Consensus Layer
+uses the ledger to validate blocks and transactions and apply them in order to
+maintain _the ledger state_. The Consensus Layer in turn needs the ledger state
+in order to determine when a node is allowed to mint a block, ie the _leader
+schedule_.
+
+The primary entrypoint is to the Consensus Layer is invoked by the
+https://github.com/input-output-hk/cardano-node in order to deploy an actual
+Cardano node on mainnet. There are also secondary entrypoints, such as the
+ledger query interface, as used by eg
+https://github.com/input-output-hk/cardano-db-sync to "follow" the net's chain.
+
+## Design Documents
+
+The following artifacts influence and/or describe the Consensus implementation.
+
+  * From IOHK researchers:
+
+      * Ouroboros BFT https://iohk.io/en/research/library/papers/ouroboros-bfta-simple-byzantine-fault-tolerant-consensus-protocol/
+
+      * Ouroboros Praos https://iohk.io/en/research/library/papers/ouroboros-praosan-adaptively-securesemi-synchronous-proof-of-stake-protocol/
+
+      * Ouroboros Genesis https://iohk.io/en/research/library/papers/ouroboros-genesiscomposable-proof-of-stake-blockchains-with-dynamic-availability/
+
+      * Internal notes, on the IOHK Google Docs
+
+  * The ledger specifications, cf https://github.com/input-output-hk/cardano-ledger-specs/blob/master/README.md, especially:
+
+      * "Shelley design specification" -> "Design Specification for Delegation
+        and Incentives in Cardano" (as of this line's latest commit)
+
+      * "Shelley ledger formal specification" -> "A Formal Specification of the
+        Cardano Ledger" (as of this line's latest commit)
+
+      * "Byron chain specification" -> "Specification of the Blockchain Layer"
+        (as of this line's latest commit)
+
+      * "Byron ledger specification" -> "A Formal Specification of the Cardano
+        Ledger (for the Byron release)" (as of this line's latest commit)
+
+  * Documents and presentations linked in this repository's
+    https://github.com/input-output-hk/ouroboros-network/README.md, especially:
+
+      * "Introduction to the design of Data Diffusion and Networking of Cardano
+        Shelley"
+
+      * "The Cardano Consensus (and Storage Layer)" (aka "The Consensus Report")
```

Comment:
You *could* create links for these.

Reply:
For the most part, I like exposing the structure of the URLs. That itself has some onboarding merit. But yeah, I can tidy the YouTube links eg.

Reply:
In such cases, I reach for [*reference-style links*](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links).

```markdown
* ["Shelley design specification"][1]

..

[1]: https://..
```

Anyway, minor and up to you :slightly_smiling_face: